roles/sap_general_preconfigure/README.md

```diff
@@ -1,341 +1,74 @@
-- _Type:_ `str`
-- _Default:_ `"{{ ((0.75 * (ansible_memtotal_mb + ansible_swaptotal_mb)) / 1024) | round | int }}"`
-
-The size of the tmpfs in GB. The formula used here is mentioned in SAP note 941735.<br>
-
-### sap_general_preconfigure_modify_etc_hosts
-- _Type:_ `bool`
-- _Default:_ `true`
-
-Set to `false` if you do not want the role to modify the `/etc/hosts` file.<br>
-
-### sap_general_preconfigure_etc_sysctl_sap_conf
-- _Type:_ `str`
-- _Default:_ `'/etc/sysctl.d/sap.conf'`
-
-The file name of the sysctl config file to be used<br>
-
-### sap_general_preconfigure_kernel_parameters
-- _Type:_ `list` with elements of type `dict`
-- _Default:_ (set by platform/environment specific variables)
-
-The Linux kernel parameters to use. By default, these are taken from the vars file.<br>
-The default for this parameter is set in the vars file which corresponds to the detected OS version.<br>
-
-Example:
-
-```yaml
-sap_general_preconfigure_kernel_parameters:
-- name: vm.max_map_count
-  value: '2147483647'
-- name: fs.aio-max-nr
-  value: '18446744073709551615'
-```
-
-### sap_general_preconfigure_max_hostname_length
-- _Type:_ `str`
-- _Default:_ `'13'`
-
-The maximum length of the hostname. See SAP note 611361.<br>
-
-### sap_hostname
-- _Type:_ `str`
-- _Default:_ `"{{ ansible_hostname }}"`
-
-The hostname to be used for updating or checking `/etc/hosts` entries.<br>
-
-### sap_domain
-- _Type:_ `str`
-- _Default:_ `"{{ ansible_domain }}"`
-
-The DNS domain name to be used for updating or checking `/etc/hosts` entries.<br>
-
-### sap_ip
-- _Type:_ `str`
-- _Default:_ `"{{ ansible_default_ipv4.address }}"`
-
-The IPV4 address to be used for updating or checking `/etc/hosts` entries.<br>
-
-### sap_general_preconfigure_db_group_name
-- _Type:_ `str`
-
-Use this variable to specify the name of the RHEL group which is used for the database processes.<br>
-If defined, it will be used to configure process limits as per step<br>
-Configuring Process Resource Limits<br>
-
-Example:
+This is general preconfigure role that used for both SAP Netweaver and SAP HANA, which have separate follow-up roles [sap_hana_preconfigure](https://github.com/sap-linuxlab/community.sap_install/tree/main/roles/sap_hana_preconfigure) and [sap_netweaver_preconfigure](https://github.com/sap-linuxlab/community.sap_install/tree/main/roles/sap_netweaver_preconfigure).
+<!-- END Description -->
 
+<!-- BEGIN Dependencies -->
+## Dependencies
+- `fedora.linux_system_roles`
+    - Roles:
+        - `selinux`
+- `community.sap_install` (This collection)
+    - Roles:
+        - `sap_maintain_etc_hosts`
+
+Install required collections by `ansible-galaxy install -vv -r meta/collection-requirements.yml`.
+<!-- END Dependencies -->
+
+<!-- BEGIN Prerequisites -->
+## Prerequisites
+
+(Red Hat specific) Ensure system is installed according to:
+- RHEL 7: SAP note 2002167, Red Hat Enterprise Linux 7.x: Installation and Upgrade, section `Installing Red Hat Enterprise Linux 7`.
```

Comment:
In collection readme it says that collection only supports RHEL 8 and 9.

Reply:
@berndfinger I took this section from original Readme. Do you want me to remove RHEL 7 line?

Reply:
@marcelmamula Yes, please remove it.

Reply:
It will be removed with new commit.